diff -Nurp a/Makefile b/Makefile
--- a/Makefile	2021-08-29 16:05:49.000000000 -0400
+++ b/Makefile	2021-12-08 13:12:35.000000000 -0500
@@ -247,85 +247,22 @@ EXTRA_CPPFLAGS+=-D_REENTRANT=1
 # We're Linux-bound anyway...
 EXTRA_CPPFLAGS+=-D_GNU_SOURCE
 
-# Backward compatibility shenanigan: before FBINK_FOR_KOBO was implemented, we assumed KOBO was the default/fallback platform.
-# Keep honoring that.
-ifndef LINUX
-ifndef CERVANTES
-ifndef LEGACY
-ifndef KINDLE
-ifndef REMARKABLE
-ifndef POCKETBOOK
-	KOBO=true
-endif
-endif
-endif
-endif
-endif
-endif
+TABULA=true
 
-# Toggle Kindle support
-ifdef KINDLE
-	TARGET_CPPFLAGS+=-DFBINK_FOR_KINDLE
-endif
-# Toggle Legacy Kindle support
-ifdef LEGACY
-	TARGET_CPPFLAGS+=-DFBINK_FOR_LEGACY
-endif
-# Toggle Bq Cervantes support
-ifdef CERVANTES
-	TARGET_CPPFLAGS+=-DFBINK_FOR_CERVANTES
-endif
-# Toggle generic Linux support
-ifdef LINUX
-	TARGET_CPPFLAGS+=-DFBINK_FOR_LINUX
-endif
-# Toggle Kobo support
-ifdef KOBO
+
+ifdef TABULA
 	TARGET_CPPFLAGS+=-DFBINK_FOR_KOBO
 endif
-# Toggle reMarkable support
-ifdef REMARKABLE
-	TARGET_CPPFLAGS+=-DFBINK_FOR_REMARKABLE
-endif
-# Toggle PocketBook support
-ifdef POCKETBOOK
-	TARGET_CPPFLAGS+=-DFBINK_FOR_POCKETBOOK
-endif
 
 # And that should definitely be honored by everything, so, add it to EXTRA_CPPFLAGS
 EXTRA_CPPFLAGS+=$(TARGET_CPPFLAGS)
 
 # A version tag...
 # NOTE: Don't redirect stderr, so we can get an idea of what happened when things go wrong...
-FBINK_VERSION:=$(shell git describe || cat VERSION)
+FBINK_VERSION:=$(file < VERSION)
 # Only use it if we got something useful...
 ifdef FBINK_VERSION
-	ifdef KINDLE
-		LIB_CFLAGS+=-DFBINK_VERSION='"$(FBINK_VERSION) for Kindle"'
-	else
-		ifdef CERVANTES
-			LIB_CFLAGS+=-DFBINK_VERSION='"$(FBINK_VERSION) for Cervantes"'
-		else
-			ifdef LINUX
-				LIB_CFLAGS+=-DFBINK_VERSION='"$(FBINK_VERSION) for Linux"'
-			else
-				ifdef KOBO
-					LIB_CFLAGS+=-DFBINK_VERSION='"$(FBINK_VERSION) for Kobo"'
-				else
-					ifdef REMARKABLE
-						LIB_CFLAGS+=-DFBINK_VERSION='"$(FBINK_VERSION) for reMarkable"'
-					else
-						ifdef POCKETBOOK
-							LIB_CFLAGS+=-DFBINK_VERSION='"$(FBINK_VERSION) for PocketBook"'
-						else
-							# NOTE: Should never happen!
-							LIB_CFLAGS+=-DFBINK_VERSION='"$(FBINK_VERSION)"'
-						endif
-					endif
-				endif
-			endif
-		endif
-	endif
+	LIB_CFLAGS+=-DFBINK_VERSION='"$(FBINK_VERSION) for Tabula"'
 else
 	# Just so we don't use an empty var down the line...
 	FBINK_VERSION:=dev
@@ -354,7 +291,7 @@ ifdef UNIBREAK
 endif
 
 # Same for libi2c, on Kobo
-ifdef KOBO
+ifdef TABULA
 	EXTRA_CPPFLAGS+=-Ilibi2c-staged/include
 	EXTRA_LDFLAGS+=-Llibi2c-staged/lib
 	I2C_LIBS:=-l:libi2c.a
@@ -415,6 +352,14 @@ endif
 
 CMD_SRCS:=fbink_cmd.c
 BTN_SRCS:=button_scan_cmd.c
+
+ifdef TABULA
+	MINIMAL=true
+	IMAGE=true
+# required to build fbink_cmd
+	BITMAP=true
+endif
+
 # Unless we're asking for a minimal build, include the other features, except for Unifont.
 ifdef MINIMAL
 	FEATURES_CPPFLAGS+=-DFBINK_MINIMAL
